src/components/Onboarding/ColumnSelectorSidebar.tsx

```diff
@@ -207,7 +207,7 @@ export default function ColumnSelectorSidebar(props: { plotArgs: Args }) {
   function handleDelete(e: React.MouseEvent<HTMLButtonElement, MouseEvent>) {
     const updatedSet = new Set(selectedColumns);
     const selectedTag = (e.currentTarget as HTMLButtonElement).className.split(
-      '_'
+      /_(.+)/
     )[1];
 
     updatedSet.delete(selectedTag);
```

Comment:
Could we rewrite this function? It's generally bad practice to update the DOM directly in a React app because the source of truth should be the React State. A good way to start would be the following:
```
function handleDelete(column: string){...}
...
onClick={() => handleDelete(column)}
```

You can delete lines 215-218 because the logic on line 249 should already do this automatically : `checked={selectedColumns.includes(col)}`